CMakeLists.txt

```diff
@@ -53,7 +53,10 @@ find_library(JPEG_LIBRARY jpeg REQUIRED)
 find_library(TIFF_LIBRARY tiff REQUIRED)
 find_library(PNG_LIBRARY png REQUIRED)
 find_library(LIBCAMERA_LIBRARY libcamera.so REQUIRED)
-message(STATUS "${LIBCAMERA_LIBRARY}")
+find_library(LIBCAMERA_BASE_LIBRARY libcamera-base.so REQUIRED)
```

Comment:
Does find_library use pkg-config? This should be provided by the libcamera.pc pkg-config so you shouldn't have to specify it manually....

If you do - it means I failed :-(

Reply:
I daresay there's a "proper" way to do it, but I claim absolutely no expertise in cmake. I'm open to offers, of course...